lightly/models/modules/masked_autoencoder_timm.py

```diff
@@ -113,7 +114,7 @@ def forward(self, input: Tensor) -> Tensor:
             Tensor with shape (batch_size, seq_length, out_dim).
 
         """
-        out = self.embed(input)
+        out: Tensor = self.embed(input)
```

Comment:
Are you sure this is necessary? `embed` has a type annotation that says it returns a `Tensor`.

Reply:
Apparently not. `mypy` was failing locally for me without this, but I cleared my cache and it works. Dropped in latest commit.